iywstbdywdiy/sheeper.py

```diff
@@ -88,24 +89,24 @@ def make_image(self, image, is_test, bounding_boxes=(), stamps=()):
             width = x2 - x1
             height = y2 - y1
 
-            resized_to_bb = stamps[i].resize((width, height), Image.ANTIALIAS)
+            resized_to_bb = stamps[i].resize((width, height), Image.LANCZOS)
 
             layer = Image.new("RGBA", image.size, (0, 0, 0, 0))
             layer.paste(resized_to_bb, box=(x1, y1))
             out = Image.alpha_composite(out, layer)
 
-        out.convert("RGB").save(output_path)
+        out.convert("RGB").save(output_path, quality=95)
```

Comment:
why not `quality=100`?

Reply:
[From the docs](http://pillow.readthedocs.io/en/5.1.x/handbook/image-file-formats.html#jpeg):
> The image quality, on a scale from 1 (worst) to 95 (best). The default is 75. **Values above 95 should be avoided**; 100 disables portions of the JPEG compression algorithm, and results in **large files with hardly any gain in image quality**.